build/Targets/References.targets

```diff
@@ -222,7 +222,7 @@
       <ItemGroup>
 
         <PackageReference Include="Moq" Version="4.2.1402.2112" />
-        <PackageReference Include="xunit" Version="2.1.0" />
+        <PackageReference Include="xunit" Version="2.2.0-beta4-build3444" />
```

Comment:
I can do some wall-clock timers.  How bad was the regression?  Was it on discovery or test runs?